librz/include/rz_project.h

```diff
@@ -24,6 +24,8 @@ typedef enum rz_project_err {
 	RZ_PROJECT_ERR_NEWER_VERSION,
 	RZ_PROJECT_ERR_INVALID_CONTENTS,
 	RZ_PROJECT_ERR_MIGRATION_FAILED,
+	RZ_PROJECT_ERR_COMPRESSION_FAILED,
+	RZ_PROJECT_ERR_MKSTEMP_FAILED,
```

Comment:
I don't think it makes too much sense to go that detailed in RzProjectErr about the reason of the failing... Whether we use mkstemp or something else is pretty much an internal detail. Right @thestr4ng3r ?

Reply:
Yes, I suggest to remove MKSTEMP error